dependencies/common/.gitignore

```diff
@@ -17,5 +17,3 @@ profvis/
 profvis*.tar.gz
 *.tar.bz
 *.tar.bz2
-plumber/
```

Comment:
We should leave this in the `.gitignore` (otherwise everyone's indexes will be needlessly dirty)